### PR TITLE
Improve SEO

### PR DESCRIPTION
**Pull request by arjunsalyan**
- [ ] Add home page meta description
- [x] Improve port-detail page title
- [ ] Add port-page meta description

https://github.com/macports/macports-webapp/issues/143

### CODE REVIEW

File: app/ports/templates/ports/portdetail.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
 {% endblock %}
 
-{% block title %}{{ req_port.name }}- Port |{% endblock %}
+{% block title %}Install {{ req_port.name }} on macOS with{% endblock %}
```
Comment (mojca) [inline app/ports/templates/ports/portdetail.html:12]: Wasn't this sentence supposed to end with `MacPorts`?

**Review by mojca — COMMENTED** [review]
I suggest to merge this.
Either in master (if still relevant) or into gsoc2020.

File: app/ports/templates/ports/portdetail.html (hunk 1 above)
Comment (arjunsalyan) [reply app/ports/templates/ports/portdetail.html:12]: > Wasn't this sentence supposed to end with `MacPorts`?

`MacPorts` is added automatically at the end by the layout file.

**Comment by arjunsalyan** [comment]
> I suggest to merge this.
> Either in master (if still relevant) or into gsoc2020.

This is no longer relevant. The title has already been modified.

**Pull request closed without merging**